### PR TITLE
JBPM-5644: Stunner - Add Catching Intermediate Error Event

### PR DESCRIPTION
**Pull request by wmedvede**

### CODE REVIEW

**Comment by wmedvede** [comment]
Hello @romartin @hasys , would you mind review this PR?
Thanks.

**Comment by hasys** [comment]
Jenkins execute full downstream build

**Comment by wmedvede** [comment]
org.kie.workbench.common.services.backend.compiler.nio.kie.KieDefaultMavenCompilerTest.buildWithPullRebaseUberfireTest

java.io.IOException: Failed to open server socket for /127.0.0.1:9418
Stacktrace

org.uberfire.java.nio.IOException: java.io.IOException: Failed to open server socket for /127.0.0.1:9418
	at java.net.PlainSocketImpl.socketBind(Native Method)
	at java.net.AbstractPlainSocketImpl.bind(AbstractPlainSocketImpl.java:387)
	at java.net.ServerSocket.bind(ServerSocket.java:375)

**Comment by wmedvede** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
Jenkins please retest this

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/resources/images/bpmn-shapes-tester-input.json [hunk 1]
```diff
@@ -190,6 +190,14 @@
         "refs": "eventSignal"
       }
     },
+    {
+      "id": "intermediateErrorCatchingEvent",
+      "name": "Catching Intermediate Event - Error",
+      "shape": {
```
Comment (romartin) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/resources/images/bpmn-shapes-tester-input.json]: minor thing - missing to add the `icon` stuff too, this way UX can see the shape and the icon in the same tester page. thx!

**Comment by romartin** [comment]
Jenkins please retest this

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/resources/images/bpmn-shapes-tester-input.json (hunk 1 above)
Comment (wmedvede) [reply kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/resources/images/bpmn-shapes-tester-input.json]: both intermediate error and end error were fixed.

**Comment by wmedvede** [comment]
org.kie.workbench.common.services.backend.compiler.nio.kie.KieDefaultMavenCompilerTest.buildWithJGitDecoratorTest
java.io.IOException: Failed to open server socket for /127.0.0.1:9418

**Comment by wmedvede** [comment]
Jenkins execute full downstream build

**Comment by romartin** [comment]
Jenkins please retest this

**Pull request merged**